dev/bots/analyze.dart

```diff
@@ -280,6 +283,39 @@ Future<void> _verifyNoMissingLicenseForExtension(String workingDirectory, String
   }
 }
 
+final RegExp _skipTestCommentPattern = RegExp(r'skip:.*?//(.*)');
+const Pattern _skipTestIntentionalPattern = '[intended]';
+final Pattern _skipTestTrackingBugPattern = RegExp(r'https+?://github.com/.*/issues/[0-9]+');
+
+Future<void> verifySkipTestComments(String workingDirectory) async {
+  final List<String> errors = <String>[];
+  assert("// foo\n}, skip: isBrowser); // this is some text\n'".contains(_skipTestCommentPattern));
+  final List<File> testFiles = await _allFiles(workingDirectory, 'dart', minimumMatches: 1500)
+      .where((File f) => f.path.endsWith('_test.dart')).toList();
```

Comment:
nit: you could remove the "toList" here and just declare testFiles as an `Iterable<File>`. That may be more efficient?

Reply:
I agree, but unfortunately, `_allFiles` returns a `Stream<File>`. I just updated this section to just iterate over the stream directly, which will hopefully be more efficient.